packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -38,11 +39,13 @@ const axiosRequestConfigToFetchRequestConfig = (requestConfig: AxiosRequestConfi
 
 export class RouterliciousRestWrapper extends RestWrapper {
     private authorizationHeader: string | undefined;
+    private token: ITokenResponse | undefined;
     private readonly restLess = new RestLessClient();
 
     constructor(
         logger: ITelemetryLogger,
         private readonly rateLimiter: RateLimiter,
+        private readonly fetchToken: TokenFetcher,
```

Comment:
I think we can cleanup a lot of the `undefined` cases here by moving `private token: ITokenResponse | undefined;` to params as `private token: ITokenResponse,`, and switching `RouterliciousRestWrapper.load` to follow the same pattern as the Orderer and Storage wrappers. like

```ts
export class RouterliciousRestWrapper extends RestWrapper {
    private readonly restLess = new RestLessClient();

    private constructor(
        logger: ITelemetryLogger,
        private readonly rateLimiter: RateLimiter,
        private token: ITokenResponse;
        private readonly fetchToken: TokenFetcher,
        private readonly getAuthorizationHeader: AuthorizationHeaderGetter,
        private readonly useRestLess: boolean,
        baseurl?: string,
        defaultQueryString: querystring.ParsedUrlQueryInput = {},
    ) {
        super(baseurl, defaultQueryString);
    }

    public static async load(
        logger: ITelemetryLogger,
        rateLimiter: RateLimiter,
        fetchToken: TokenFetcher,
        getAuthorizationHeader: AuthorizationHeaderGetter,
        useRestLess: boolean,
        baseurl?: string,
        defaultQueryString: querystring.ParsedUrlQueryInput = {},
    ): Promise<RouterliciousRestWrapper> {
        const token = await this.fetchToken();
        return new RouterliciousRestWrapper(
            logger: ITelemetryLogger,
            rateLimiter: RateLimiter,
            token,
            fetchToken: TokenFetcher,
            getAuthorizationHeader: AuthorizationHeaderGetter,
            useRestLess: boolean,
            baseurl?: string,
            defaultQueryString: querystring.ParsedUrlQueryInput = {},
        );
    }
   // ...
}
```